core/http/Cargo.toml

```diff
@@ -22,16 +22,16 @@ private-cookies = ["cookie/secure"]
 smallvec = "0.6"
 percent-encoding = "1"
 hyper = { version = "0.10.13", default-features = false }
-time = "0.1"
+time = "0.2.4"
 indexmap = "1.0"
-rustls = { version = "0.14", optional = true }
+rustls = { version = "0.16", optional = true }
```

Comment:
Changing these versions is a breaking change due to the `ring` dependency. Backporting this to 0.4 would instead require backporting the `reset_delta` to the cookie 0.11 series - @SergioBenitez, is that likely or unlikely to be accepted?

Reply:
Happy to accept a backport to cookie 0.11 and subsequently to 0.4.

Reply:
I created a PR https://github.com/SergioBenitez/cookie-rs/pull/135

Reply:
I have updated cookies to 0.11.2 now